catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/CreateRequestImpl.java

```diff
@@ -60,8 +65,23 @@ public CreateRequestImpl(List<Metacard> metacards) {
      *            the properties
      */
     public CreateRequestImpl(List<Metacard> metacards, Map<String, Serializable> properties) {
+        this(metacards, properties, new HashSet<>());
+    }
+
+    public CreateRequestImpl(List<Metacard> metacards, Map<String, Serializable> properties,
+            Set<String> destinations) {
         super(properties);
         this.metacards = metacards;
+        if (destinations != null) {
+            this.destinations = destinations;
+        } else {
+            this.destinations = new HashSet<>();
```

Comment:
Why not just always initialize `destinations` to `new HashSet<>()` and avoid the else?